pfsspy/utils.py

```diff
@@ -100,18 +100,32 @@ def carr_cea_wcs_header(dtime, shape):
     shape : tuple
         Map shape. The first entry should be number of points in longitude, the
         second in latitude.
+    map_center_longitude : astropy.Quantity
+        Change the world coordinate longitude of the central image pixel to allow
+        for different roll angles of the Carrington map. Default to 0 deg. Must
+        be supplied with units of `astropy.units.deg`
 
     References
     ----------
     .. [1] W. T. Thompson, "Coordinate systems for solar image data",
        https://doi.org/10.1051/0004-6361:20054262
     """
+    # Check map_center_longitude has units of degrees
+    assert hasattr(map_center_longitude, "unit"), (
+        f"Keyword `map_center_longitude`={map_center_longitude} has no "
+        + f"`astropy.units` attribute, must have units and be `astropy.units.deg`."
+    )
+    assert map_center_longitude.unit == u.deg, (
+        f"Keyword `map_center_longitude`={map_center_longitude} does not have "
+        +f"units of `astropy.units.deg`"
+    )
```

Comment:
Instead of doing these checks manually, you can use https://docs.astropy.org/en/stable/api/astropy.units.quantity_input.html

Reply:
Change made with https://github.com/dstansby/pfsspy/pull/358/commits/2e015ab6acc7326ba7ac8a9ca638d766b4ed54a7